cmd/kube-proxy/app/server.go

```diff
@@ -410,7 +410,7 @@ func createClients(config componentconfig.ClientConnectionConfiguration, masterO
 	kubeConfig.ContentType = config.ContentType
 	kubeConfig.QPS = config.QPS
 	//TODO make config struct use int instead of int32?
```

Comment:
Is it worthy fixing this TODO? There might be tens of files to change.

Reply:
this seems to be breaking CI batching https://github.com/kubernetes/test-infra/issues/5518